### PR TITLE
Added a status API file and healthcheck endpoint

### PR DESCRIPTION
**Pull request by robbiethegeek**
Move the current api to a routes folder and rename application.

### CODE REVIEW

**Comment by akainic** [comment]
LGTM, but probably want to have @alexose or @awong-dev review as well!

Now that we have this healthcheck endpoint, that's what we'll ping to make sure our app is up and running? How will that work?

**Comment by robbiethegeek** [comment]
@akainic we can use this as a way for the load balancer to know that the application is up, we can also on the front end have a quick fetch of this url to validate that the node app is up and running and have that tell the end user they are one step closer to the being able to submit if the response is 200 OK.

File: src/server/routes/status.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+function returnRouter() {
+  const router = require('express').Router(); // eslint-disable-line
+  router.get('/', (req, res) => {
+    res.status(501).end();
```
Comment (alexose) [inline src/server/routes/status.js]: Placeholder for discoverability :)

**Comment by alexose** [comment]
Seems perfectly reasonable to me.  👍

**Pull request merged**